assets/src/components/map.tsx

```diff
@@ -330,156 +184,56 @@ export const autoCenter = (
   }
 }
 
-const recenterControl = (
-  setShouldAutoCenter: (shouldAutoCenter: boolean) => void,
-  controlOptions: Leaflet.ControlOptions
-): Leaflet.Control => {
-  const RecenterControl = Leaflet.Control.extend({
-    options: controlOptions,
-    onAdd: () => {
-      const container: HTMLElement = Leaflet.DomUtil.create(
-        "div",
-        "leaflet-bar leaflet-control m-vehicle-map__recenter-button"
-      )
-      const link: HTMLLinkElement = Leaflet.DomUtil.create(
-        "a",
-        "",
-        container
-      ) as HTMLLinkElement
-      link.innerHTML = `<svg
-        height="30"
-        viewBox="-5 -2 36 36"
-        width="30"
-        xmlns="http://www.w3.org/2000/svg"
+const RecenterControl = ({
+  turnOnAutoCenter,
+}: {
+  turnOnAutoCenter: () => void
+}) => (
+  <Control position="topright">
+    <div className="leaflet-bar m-vehicle-map__recenter-button">
+      <a
+        href="#"
+        title="Recenter map"
+        role="button"
+        aria-label="Recenter map"
+        onClick={(e) => {
```

Comment:
Codecov is dinging this. I don't think it's the most crucial thing, but would it be relatively straightforward that clicking this does something, like e.g. the `autoCenteringClass` changes (even though that's kind of a second-order effect).

Reply:
👍 I just pushed a test that checks the class and the map center.